### ISSUE TITLE
Upgrade polymake to 4.9

### ISSUE THREAD

**Issue by mkoeppe** [sagemath/sage#34866]
<div id="comment:0"></div>

https://polymake.org/doku.php/news/release_4_8
https://polymake.org/doku.php/news/release_4_9

Component: **packages: optional**

Author: **Matthias Koeppe**

_Issue created by migration from https://trac.sagemath.org/ticket/34866_

**Comment by mkoeppe** [comment]
Branch: **[u/mkoeppe/upgrade_polymake_to_4_8](https://github.com/sagemath/sagetrac-mirror/tree/u/mkoeppe/upgrade_polymake_to_4_8)**

**Comment by mkoeppe** [comment]
<div id="comment:2" align="right">comment:2</div>


```
sage -t --random-seed=65134114044879811697120258620233978088 src/sage/interfaces/polymake.py
**********************************************************************
File "src/sage/interfaces/polymake.py", line 754, in sage.interfaces.polymake.PolymakeAbstract.application
Failed example:
    q.VERY_AMPLE                  # optional - jupymake
Exception raised:
    Traceback (most recent call last):
      File "/Users/mkoeppe/s/sage/sage-rebasing/worktree-rebase/src/sage/interfaces/interface.py", line 735, in __init__
        self._name = parent._create(value, name=name)
      File "/Users/mkoeppe/s/sage/sage-rebasing/worktree-rebase/src/sage/interfaces/polymake.py", line 517, in _create
        self.set(name, value)
      File "/Users/mkoeppe/s/sage/sage-rebasing/worktree-rebase/src/sage/interfaces/polymake.py", line 606, in set
        self.eval(cmd)
      File "/Users/mkoeppe/s/sage/sage-rebasing/worktree-rebase/src/sage/interfaces/polymake.py", line 2029, in eval
        raise PolymakeError(error)
    sage.interfaces.polymake.PolymakeError: no more rules available to compute 'VERY_AMPLE'
```

---
New commits:
<table><tr><td><a href="https://github.com/sagemath/sagetrac-mirror/commit/b0ef9fbf12f9861f4a0e35f7552e197a95f218cc"><code>b0ef9fb</code></a></td><td><code>build/pkgs/polymake: Update to 4.8</code></td></tr><tr><td><a href="https://github.com/sagemath/sagetrac-mirror/commit/362c22d4734461bd6eb79660fb2b9286f55084a4"><code>362c22d</code></a></td><td><code>build/pkgs/polymake/spkg-install.in: Add -I directives to CXXFLAGS</code></td></tr><tr><td><a href="https://github.com/sagemath/sagetrac-mirror/commit/dcbee984ac0d21480b12a9830a5aef9382ee2c61"><code>dcbee98</code></a></td><td><code>build/pkgs/polymake/dependencies: Add lrslib (needed for sympol)</code></td></tr></table>

**Comment by mkoeppe** [comment]
Author: **Matthias Koeppe**

**Comment by mkoeppe** [comment]
Commit: **[`dcbee98`](https://github.com/sagemath/sagetrac-mirror/commit/dcbee984ac0d21480b12a9830a5aef9382ee2c61)**

**Comment by mkoeppe** [comment]
<div id="comment:3" align="right">comment:3</div>

Detecting bliss, libnormaliz, soplex fails

**Comment by mkoeppe** [comment]
Removed branch from issue description; replaced by PR #35085.

**Comment by mkoeppe** [comment]
> Detecting bliss, libnormaliz, soplex fails

This seems to be fixed in 4.9 (and soplex is gone)
